styles/style.css

```diff
@@ -103,6 +128,7 @@ p {
 .content {
   display: flex;
   flex-direction: column;
+  margin:0%;
```

Comment:
One more thing, I just saw that your main content is aligned to the left and there are lots of spaces on the right. Try removing this margin and it should work

Reply:
Thanks Amellie, removed the margin